<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Art Academy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	font-family: url(:/fonts/OpenSans-Regular.ttf);
	font-size: 35px;
}
#main_frame {
	background-color: #fafafa;
}

#menu_frame {
	background-color: #b71c1c;
}

QRadioButton {
	background-color: none;
	border: none;
	color: white;
	font-weight: bold;
	font-size: 16px;
	padding: 25px;
	margin-left: -20px;
}

QRadioButton:hover {
	background-color: #c62828;
}

QRadioButton:checked {
	background-color: #ef5350;
}

QRadioButton::indicator {
	border: none;
}
QLabel#logo {
	background-color: #689f38;
	color: #212121;
}
QFrame#header_frame {
	background-color: #f57c00;
}

#pages {
	background-color: #fafafa;
}

#new_user {
	background-color: #b71c1c;
	border: none;
	padding: 10px;
	color: #fafafa;
	font-size: 14px;
	width: 20px;
	height: 20px;
}


#new_user:hover {
	background-color: #c62828;
}

#search_box {
	background-color: #fafafa;
	border: 2px solid #b71c1c;
	padding: 10px 8px;
	font-size: 14px;
	width: 20px;
	height: 20px;
}

#search_button {
	border: 2px soild #b71c1c;
	background-color: #b71c1c;
	width: 50px;
	height: 24px;
	padding: 10px;
}
#search_button:hover {
	background-color: #c62828;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,100">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="menu_frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,10,10,10,10,80,10">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="logo_label">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="logo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans Extrabold</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Art Academy</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="user_layout">
         <item>
          <widget class="QRadioButton" name="users">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>    Users</string>
           </property>
           <property name="icon">
            <iconset resource="resources/res.qrc">
             <normaloff>:/icons/user.png</normaloff>:/icons/user.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="profile_layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="profiles">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>    Profiles</string>
           </property>
           <property name="icon">
            <iconset resource="resources/res.qrc">
             <normaloff>:/icons/profile.png</normaloff>:/icons/profile.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="payment_layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="payments">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>    Payments</string>
           </property>
           <property name="icon">
            <iconset resource="resources/res.qrc">
             <normaloff>:/icons/payment.png</normaloff>:/icons/payment.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="send_mail_layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="send_email">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>    Send Email</string>
           </property>
           <property name="icon">
            <iconset resource="resources/res.qrc">
             <normaloff>:/icons/email.png</normaloff>:/icons/email.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="setting_layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="help">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>    Help</string>
           </property>
           <property name="icon">
            <iconset resource="resources/res.qrc">
             <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="main_frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="header_frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="header_text">
            <property name="text">
             <string>User Managment</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="pages">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="users_page">
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,20">
           <item>
            <widget class="QFrame" name="user_top">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,0,5,3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLineEdit" name="search_box">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>Username or id or mobile number ...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="search_button">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources/res.qrc">
                  <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="new_user">
                <property name="text">
                 <string>New User</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="table_widget">
             <column>
              <property name="text">
               <string>Username</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Password</string>
              </property>
              <property name="textAlignment">
               <set>AlignJustify|AlignVCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Edit</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Delete</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="profiles_page">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>30</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Profile page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="payments_page"/>
         <widget class="QWidget" name="send_mail_page"/>
         <widget class="QWidget" name="help_page"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
